File: Codigo_Fonte_c/Lista 4/lista4_8.c
#include <stdio.h>

int main(){
	int matriz[4][4];
	int i, j, sum = 0;
	for (i = 0; i < 4; i++){
		for (j = 0;j < 4; j++){
			scanf("%d", &matriz[i][j]);
		}
	}
	
	for (i = 0; i < 4; i++){
		for (j = 0; j < 4; j++){
			if ((i + j) == 4 - 1){
				sum = sum + matriz[i][j];
				}
			}
	}
	printf("A soma da diagonal secundaria: %d\n", sum);
	return 0;
}

File: Codigo_Fonte_pascal/lista4/lista4_8.pas
program diagonal_secundaria;

var
   A : array[1..4, 1..4] of integer;
   i, j, sum : integer;

begin
     sum := 0;
     for i := 1 to 4 do
          for j := 1 to 4 do
              readln(A[i, j]);

     for i := 1 to 4 do
     begin
          for j := 1 to 4 do
          begin
               if (i + j) = 4 + 1 then
                  sum := sum + A[i, j];
          end;

     end;
     writeln('A soma da diagonal secundaria: ', sum);
     readln;
end.
